javascript/javascript2/week2/movies/movies.js

```diff
@@ -0,0 +1,67 @@
Junebug", "year": 2005, "rating": 7, "votes": 19931, "running_times": 6360 }, { "title": "Jungle", "year": 2017, "rating": 6.7, "votes": 16117, "running_times": 6900 }, { "title": "Jungle 2 Jungle", "year": 1997, "rating": 5.1, "votes": 19645, "running_times": 6300 }, { "title": "Jungle Fever", "year": 1991, "rating": 6.5, "votes": 14456, "running_times": 7920 }, { "title": "Junior", "year": 1994, "rating": 4.6, "votes": 57432, "running_times": 6540 }, { "title": "Juno", "year": 2007, "rating": 7.5, "votes": 442969, "running_times": 5760 }, { "title": "Jupiter Ascending", "year": 2015, "rating": 5.3, "votes": 155246, "running_times": 7620 }, { "title": "Jurassic Park", "year": 1993, "rating": 8.1, "votes": 695981, "running_times": 7620 }, { "title": "Jurassic Park III", "year": 2001, "rating": 5.9, "votes": 242697, "running_times": 5520 }, { "title": "Jurassic World", "year": 2015, "rating": 7, "votes": 476644, "running_times": 7440 }, { "title": "Just Before I Go", "year": 2014, "rating": 6.4, "votes": 15316, "running_times": 5700 }, { "title": "Just Cause", "year": 1995, "rating": 6.4, "votes": 21147, "running_times": 6120 }, { "title": "Just Friends", "year": 2005, "rating": 6.2, "votes": 89395, "running_times": 5760 }, { "title": "Just Go with It", "year": 2011, "rating": 6.4, "votes": 187549, "running_times": 7020 }, { "title": "Just Like Heaven", "year": 2005, "rating": 6.7, "votes": 88168, "running_times": 5700 }, { "title": "Just Married", "year": 2003, "rating": 5.5, "votes": 61009, "running_times": 5700 }, { "title": "Just My Luck", "year": 2006, "rating": 5.4, "votes": 47721, "running_times": 6180 }, { "title": "Just One of the Guys", "year": 1985, "rating": 6.5, "votes": 10976, "running_times": 6000 }, { "title": "Just Visiting", "year": 2001, "rating": 5.8, "votes": 16255, "running_times": 5280 }, { "title": "Just Wright", "year": 2010, "rating": 5.9, "votes": 11239, "running_times": 6000 }, { "title": "Justice League", "year": 2017, "rating": 7.2, "votes": 141370, "running_times": 7200 }, { "title": "Justin Bieber: Never Say Never", "year": 2011, "rating": 1.6, "votes": 74720, "running_times": 6900 }, { "title": "K-19: The Widowmaker", "year": 2002, "rating": 6.7, "votes": 52505, "running_times": 8280 }, { "title": "K-9", "year": 1989, "rating": 6, "votes": 27515, "running_times": 6060 }, { "title": "K-PAX", "year": 2001, "rating": 7.4, "votes": 160033, "running_times": 7200 }, { "title": "Kabhi Alvida Naa Kehna", "year": 2006, "rating": 6.1, "votes": 15518, "running_times": 11580 }, { "title": "Kabhi Haan Kabhi Naa", "year": 1994, "rating": 8, "votes": 14320, "running_times": 9480 }, { "title": "Kabhi Khushi Kabhie Gham...", "year": 2001, "rating": 7.5, "votes": 35341, "running_times": 12600 }, { "title": "Kaboom", "year": 2010, "rating": 5.7, "votes": 11187, "running_times": 5160 }, { "title": "Kahaani", "year": 2012, "rating": 8.2, "votes": 46148, "running_times": 7320 }, { "title": "Kaho Naa... Pyaar Hai", "year": 2000, "rating"
```

Comment:
you don't need to use map here, reduce will work on any array and the input doesn't need to match the return value, the trick is to set an initial value, if you inputs is an array of objects, but you want to return a number, just set a number as the initial value and then it will be the first value of the accumulator

```
const array1 = [1, 2, 3, 4];
const reducer = (accumulator, currentValue) => accumulator + currentValue;

// 5 + 1 + 2 + 3 + 4
console.log(array1.reduce(reducer, 5));
// expected output: 15
```
this example shows the use of an initial value